geo/src/main/java/org/odk/collect/geo/GeoPointActivity.java

```diff
@@ -245,7 +231,7 @@ private void returnLocation() {
         if (location != null) {
             Intent i = new Intent();
 
-            i.putExtra(FormEntryActivity.ANSWER_KEY, getResultStringForLocation(location));
+            i.putExtra("value", getResultStringForLocation(location));
```

Comment:
This is a little strange. We can't reference `FormEntryActivity.ANSWER_KEY` from `geo` so need to just use an inline value. IMO, the `GeoPointActivity` should really be defining the intent it responds with, but this would buck the established pattern that `"value"` is used as the key for all these kinds of Activity objects.